### PR TITLE
Upgrade bazel to 2.2.0.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/toktok/toktok-stack/43)

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -1,37 +1,20 @@
...
-    - echo "build --config=linux" | sudo tee -a /etc/bazel.bazelrc
-    - echo "build --config=gcc" | sudo tee -a /etc/bazel.bazelrc
-    - echo "build --config=remote" | sudo tee -a /etc/bazel.bazelrc
-    - git submodule update --jobs=8 --init --depth=5 --recursive
+    - wget https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/ffmpeg/3.4.1-1.4/ffmpeg-3.4.1-1.4-linux-x86_64.jar -O third_party/javacpp/ffmpeg/jar/ffmpeg-3.4.1-1.4-linux-x86_64.jar
+    - wget https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/opencv/3.4.0-1.4/opencv-3.4.0-1.4-linux-x86_64.jar -O third_party/javacpp/opencv/jar/opencv-3.4.0-1.4-linux-x86_64.jar
+    - echo "build --jobs=50 --curses=no --verbose_failures" | tee ~/.bazelrc
+    - echo "build --config=linux" | tee -a ~/.bazelrc
+    - echo "build --config=clang" | tee -a ~/.bazelrc
+    - echo "build --config=remote" | tee -a ~/.bazelrc
```
Comment (robinlinden) [inline .cirrus.yml]: Isn't this currently broken?

Comment (iphydf) [reply .cirrus.yml]: Yep, removed.

File: third_party/haskell/BUILD.splitmix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    src_strip_prefix = "src-compat",
+    version = packages[PACKAGE_NAME].version,
+    visibility = ["//visibility:public"],
+    deps = [
+        hazel_library("base"),
+    ],
+)
+
+haskell_library(
+    name = "splitmix",
```
Comment (robinlinden) [inline third_party/haskell/BUILD.splitmix]: PACKAGE_NAME?

Comment (iphydf) [reply third_party/haskell/BUILD.splitmix]: Thanks, fixed.

**Pull request merged**